src/chaosinventory/settings.py

```diff
@@ -1,35 +1,24 @@
-"""
-Django settings for chaosinventory project.
-
-Generated by 'django-admin startproject' using Django 3.1.4.
+import configparser
+import os
+from pathlib import Path
 
-For more information on this file, see
-https://docs.djangoproject.com/en/3.1/topics/settings/
+config = configparser.RawConfigParser()
 
-For the full list of settings and their values, see
-https://docs.djangoproject.com/en/3.1/ref/settings/
-"""
+if 'CHAOSINVENTORY_CONFIG_FILE' in os.environ:
+    config.read_file(open(os.environ.get('CHAOSINVENTORY_CONFIG_FILE'), encoding='utf-8'))
+else:
+    config.read(['/etc/chaosinventory/chaosinventory.cfg', 'chaosinventory.cfg'], encoding='utf-8')
```

Comment:
How do I read this order? Is the file in the same directory read first or the file from etc?

Reply:
I think the config files will be checked in the following order:
1. the file at the path `CHAOSINVENTORY_CONFIG_FILE`
2. `/etc/chaosinventory/chaosinventory.cfg`
3. `chaosinventory.cfg`

Reply:
2. and 3. were swapped as discussed on 2020-01-29